crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1017,6 +1019,47 @@ impl Client {
         LoginBuilder::new_password(self.clone(), id, password.to_owned())
     }
 
+    /// Login to the server with a custom login type
+    ///
+    /// # Arguments
+    ///
+    /// * `login_type` - Identifier of the custom login type, e.g. `org.matrix.login.jwt`
+    ///
+    /// * `data` - The additional data which should be attached to the login request.
+    ///
+    /// ```no_run
+    /// # use futures::executor::block_on;
+    /// # use url::Url;
+    /// # use serde_json::json;
+    /// # let homeserver = Url::parse("http://example.com").unwrap();
+    /// # block_on(async {
+    /// use matrix_sdk::Client;
+    ///
+    /// let client = Client::new(homeserver).await?;
+    /// let user = "example";
+    ///
+    /// let response = client
+    ///     .login_custom(
+    ///         "org.matrix.login.jwt",
+    ///         [("token".to_string(), json!("jwt_token_content"))].into_iter().collect(),
```

Comment:
For consistency, `to_owned` instead of `to_string`. Also using `impl From<&str> for Value` there is no need to import `json!`.

```suggestion
    ///         [("token".to_owned(), "jwt_token_content".into())].into_iter().collect(),
```

Reply:
Thank you. Done.